### PR TITLE
Prevent Functional Tests from running on a PR from a fork

### PR DESCRIPTION
**Pull request by bkendall**
### Description

This stops Travis from running the functional tests on a PR from a fork. Because `push` events can only happen on this repository, they are always safe. Checking that the PR repo (`head_repo`, when set) is the same as the repo (`repo`) will ensure that the functional test is run only when they are the same.

There's a more convoluted solution to making the functional tests work on PRs, but I don't believe we have the time to make it :)

Also, I ran `prettier` on the file

### Tested

By making this PR from a fork, and the functional test is not run (the `pr` test below from travis passed because the unit tests passed).

### CODE REVIEW

**Comment by bkendall** [comment]
Because I pushed the same commit to a branch on `firebase/firebase-tools`, `travis-ci/push` was run as well.

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,20 @@
 language: node_js
 node_js:
-- '6'
-- '8'
-- '10'
+  - "6"
+  - "8"
+  - "10"
+before_script:
+  - env
```
[the remark this answers is not in the document]
Comment (bkendall) [reply .travis.yml]: unneeded - left it it from some debugging. thanks for catching that.

**Comment by laurenzlong** [comment]
Merged to unblock https://github.com/firebase/firebase-tools/pull/1047

**Comment by bkendall** [comment]
Thanks! 👍

**Pull request merged**